### PR TITLE
Move GraphicsOptions parameter

### PR DESCRIPTION
**Pull request by JimBobSquarePants**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
This PR reorders the parameters of any methods with the parameter `GraphicsOptions` or `TextGraphicsOptions` to make that parameter first.

Based on the suggestion by @vpenades 

https://gitter.im/ImageSharp/General?at=5aa8f2aac3c5f8b90d6df801

### CODE REVIEW

File: src/ImageSharp.Drawing/Processing/Drawing/DrawBezierExtensions.cs [hunk 1]
```diff
@@ -59,27 +59,27 @@ public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImagePro
...
         /// </summary>
         /// <typeparam name="TPixel">The type of the color.</typeparam>
         /// <param name="source">The image this method extends.</param>
+        /// <param name="options">The options.</param>
         /// <param name="pen">The pen.</param>
         /// <param name="points">The points.</param>
-        /// <param name="options">The options.</param>
         /// <returns>The <see cref="Image{TPixel}"/>.</returns>
-        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, IPen<TPixel> pen, PointF[] points, GraphicsOptions options)
+        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, GraphicsOptions options, IPen<TPixel> pen, PointF[] points)
```
Comment (tocsoft) [inline src/ImageSharp.Drawing/Processing/Drawing/DrawBezierExtensions.cs]: we will want to make these arrays `param` properties, this is true for all the extension methods ending with arrays

Comment (vpenades) [reply src/ImageSharp.Drawing/Processing/Drawing/DrawBezierExtensions.cs]: Indeed, using `, params PointF[]` is the whole point of this reordering

Comment (JimBobSquarePants) [reply src/ImageSharp.Drawing/Processing/Drawing/DrawBezierExtensions.cs]: Haha new I'd missed something 😄 Updated.

**Comment by JimBobSquarePants** [comment]
I'm gonna merge this now as I got the params stuff in.

**Pull request merged**